blackbox/docs/configuration.txt

```diff
@@ -701,6 +701,23 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``0s`` (disabled)
+  | *Runtime:*  ``yes``
+
+  This setting specifies the period when jobs are stored in the ``sys.jobs_log``
+  table. It periodically clears entries that are completed and older than the
+  specified time interval. This setting overrides ``jobs_log_size`` and logs a
+  warning message if both settings are set explicitly. If the value is set to
```

Comment:
I removed the warning with my changes. Re-add it or change the docs so that expiration has higher prio?